package.json

```diff
@@ -20,5 +22,38 @@
     "grunt-css-selectors": "^1.1.0",
     "grunt-pure-grids": "^1.0.0",
     "grunt-stripmq": "0.0.6"
-  }
+  },
+  "description": "Pure is a ridiculously tiny CSS library you can use to start any web project.",
+  "bugs": {
+    "url": "https://github.com/yahoo/pure/issues"
+  },
+  "homepage": "https://github.com/yahoo/pure",
+  "main": "README.md",
+  "dependencies": {
+    "grunt": "^0.4.5",
+    "grunt-cli": "^0.1.13",
+    "grunt-contrib-csslint": "^0.2.0",
+    "grunt-contrib-compress": "^0.13.0",
+    "bower": "^1.3.12",
```

Comment:
Especially with `bower` in here, otherwise this package will be enormous when someone goes to install it.